cluster/volume.go

```diff
@@ -8,3 +8,42 @@ type Volume struct {
 
 	Engine *Engine
 }
+
+// Volumes represents an array of volumes
+type Volumes []*Volume
+
+// Get returns a volume using it's ID or Name
+func (volumes Volumes) Get(name string) *Volume {
+	// Abort immediately if the name is empty.
+	if len(name) == 0 {
+		return nil
+	}
+
+	candidates := []*Volume{}
+
+	// Match name, /name or engine/name.
```

Comment:
The comment doesn't match code. The logic of this function is a little confusing to me. For example why 'size >1' is not a valid case. I'd suggest add comment to clarify.